synapse/models/auth.py

```diff
@@ -1,14 +1,60 @@
+import hashlib
+
+import synapse.lib.types as s_types
+
+class Passwd(s_types.Str):
+
+    def norm(self, valu):
+        retn = s_types.Str.norm(self, valu)
```

Comment:
Since strip is now defaulting to true; would this implicitly strip whitespace off of a password string that is otherwise important to retain?